common

```diff
@@ -243,7 +243,7 @@ dm_name_for_devnode() {
   else
     # don't leave the caller hanging, just print the original name
     # along with the failure.
-    print '%s' "$1"
+    printf '%s' "$1"
```

Comment:
It seems we can remove this whole line, as the line below already prints an clear error message.